corehq/apps/callcenter/tests/test_utils.py

```diff
@@ -462,7 +462,7 @@ def test_is_midnight_for_domain(self):
 
 
 @es_test(requires=[domain_adapter])
-class CallCenterDomainTest(SimpleTestCase):
+class CallCenterDomainTest(TestCase):
```

Comment:
why does this need DB access now?

Reply:
When we are trying to index domain, it is calling `form_python` which queries Subscriptions model.

Reply:
That's unfortunate since those are optional fields

Reply:
I guess I can mock the call and change test back to `SimpleTestCase`.